lib/iris/experimental/ugrid/__init__.py

```diff
@@ -8,4029 +8,29 @@
 Infra-structure for unstructured mesh support, based on
 CF UGRID Conventions (v1.0), https://ugrid-conventions.github.io/ugrid-conventions/
 
+.. note::
+
+    For the docstring of :const:`PARSE_UGRID_ON_LOAD`: see the original
+    definition at :const:`iris.experimental.ugrid.load.PARSE_UGRID_ON_LOAD`.
+
 """
+# Configure the logger.
+from ...config import get_logger
 
-from abc import ABC, abstractmethod
-from collections import namedtuple
-from collections.abc import Iterable
-from contextlib import contextmanager
-from functools import wraps
-from itertools import groupby
-import logging
-from pathlib import Path
-import re
-import threading
+logger = get_logger(__name__, fmt="[%(cls)s.%(funcName)s]")
```

Comment:
@trexfeathers This isn't quite right... but I'm tempted to bank this as is, and we fix it in a separate PR.